sdk/python/feast/infra/offline_stores/snowflake.py

```diff
@@ -470,20 +470,9 @@ def _to_df_internal(self, timeout: Optional[int] = None) -> pd.DataFrame:
         return df
 
     def _to_arrow_internal(self, timeout: Optional[int] = None) -> pyarrow.Table:
-        pa_table = execute_snowflake_statement(
+        return execute_snowflake_statement(
             self.snowflake_conn, self.to_sql()
-        ).fetch_arrow_all(force_return_table=False)
```

Comment:
This was a bug, right? Wouldn't this already raise an exception for <3.7 versions?